content/docs/cosidriver/installation/configuration_file.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Configuration File
+linktitle: Configuration File
+weight: 1
+Description: Description of configuration file for ObjectScale
+---
+
+> **Notational Conventions**
+>
+> The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
+
+## Dell COSI Driver Configuration Schema
+
+This configuration file is used to specify the settings for the Dell COSI Driver, which is responsible for managing connections to the Dell ObjectScale platform. The configuration file is written in YAML format and based on the JSON schema and adheres to it's specification.
+
+YAML files can have comments, which are lines in the file that begin with the `#` character. Comments can be used to provide context and explanations for the data in the file, and they are ignored by parsers when reading the YAML data.
+
+## Configuration file example
+
+```yaml
+# This is example of configuration file. You MUST edit it before applying!
```

Comment:
This is an example of **a** configuration file. You MUST edit the file before using it in your environment.